Remove dependencies on deprecated cFE elements

With:
`cfe/CMakeLists.txt:add_definitions("-D_ENHANCED_BUILD_ -DCFE_OMIT_DEPRECATED_6_6")`
build errors:

```
/home/jhageman/cFS/cFS-dev/apps/sch_lab/fsw/src/sch_lab_app.c: In function ‘SCH_Lab_AppMain’:
/home/jhageman/cFS/cFS-dev/apps/sch_lab/fsw/src/sch_lab_app.c:75:34: error: ‘CFE_ES_APP_RUN’ undeclared (first use in this function)
     uint32           RunStatus = CFE_ES_APP_RUN;
```

--- a/fsw/src/sch_lab_app.c
+++ b/fsw/src/sch_lab_app.c
@@ -72,7 +72,7 @@ void SCH_Lab_AppMain(void)
     int              i;
     int32            rtnStat;
     uint32           SCH_OneHzPktsRcvd = 0;
-    uint32           RunStatus = CFE_ES_APP_RUN;
+    uint32           RunStatus = CFE_ES_RunStatus_APP_RUN;
     CFE_SB_PipeId_t  SCH_CmdPipe;
 
     CFE_ES_PerfLogEntry(SCH_MAIN_TASK_PERF_ID);
@@ -88,7 +88,7 @@ void SCH_Lab_AppMain(void)
          {   
               CFE_SB_InitMsg(&SCH_CmdHeaderTable[i],
                               SCH_LAB_ScheduleTable[i].MessageID,
-                                sizeof(CFE_SB_CmdHdr_t), TRUE);
+                                sizeof(CFE_SB_CmdHdr_t), true);
          } 
          else
          {
@@ -116,7 +116,7 @@ void SCH_Lab_AppMain(void)
                 SCH_LAB_MISSION_REV);    
 
     /* Loop Forever */
-    while (CFE_ES_RunLoop(&RunStatus) == TRUE)
+    while (CFE_ES_RunLoop(&RunStatus) == true)
     {
         CFE_ES_PerfLogExit(SCH_MAIN_TASK_PERF_ID);
 

